gninasrc/CMakeLists.txt

```diff
@@ -89,6 +89,8 @@ gninavis/cnn_visualization.cpp
 
 #test
 file(GLOB TEST_SRCS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} test/*.c*)
+file(GLOB DREAM_SRCS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} gninadream/*.c*)
```

Comment:
globing is considered a cmake anti-pattern, although I agree it's much more convenient than explicitly writing out source files.